src/tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+# *******************************************************************************
+# Copyright (c) 2025 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+# *******************************************************************************
+def pytest_addoption(parser):
+    """Add custom command line options to pytest"""
+    parser.addoption(
+        "--repo-tests",
```

Comment:
Not sure why this has a tests suffix?

```suggestion
        "--repo",
```